src/components/ui/coaching-sessions/coaching-session-title.tsx

```diff
@@ -49,10 +55,40 @@ const CoachingSessionTitle: React.FC<{
 
   const displayTitle = sessionTitle?.title || defaultSessionTitle().title;
 
+  // Helper to get presence by role
+  const getPresenceByRole = (role: 'coach' | 'coachee'): UserPresence | undefined => {
+    if (!presenceState) return undefined;
+    return Array.from(presenceState.users.values()).find(u => u.role === role);
+  };
+
+  // Enhanced title rendering with presence indicators
+  const renderTitleWithPresence = () => {
+    if (!sessionTitle || !currentCoachingRelationship) return displayTitle;
+    
+    const coachPresence = getPresenceByRole('coach');
+    const coacheePresence = getPresenceByRole('coachee');
+    
+    // Parse the existing title format: "Coach Name <> Coachee Name"
+    const parts = displayTitle.split(' <> ');
```

Comment:
Missing Null Checks

```typescript
const parts = displayTitle.split(' <> ');
```
Issue: Could throw if displayTitle is null/undefined
Suggestion: Add null check before split operation, also update to '/' as mentioned in another comment